WooCommerce/Classes/Authentication/Store Creation/Profiler/Features/StoreCreationFeaturesQuestionOptions.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+extension StoreCreationFeaturesQuestionViewModel {
+    // TODO: 10386 Align with Android and send these via tracks
+    enum Feature: String, CaseIterable {
+        case salesAndAnalyticsReports = "sales-and-analytics-reports"
+        case productManagementAndInventoryTracking = "product-management-and-inventory-tracking"
+        case flexibleAndSecurePaymentOptions = "flexible-and-secure-payment-options"
+        case inPersonPayment = "in-person-payment"
+        case abilityToScaleAsBusinessGrows = "ability-to-scale-as-business-grows"
+        case customisationOptionForStoreDesign = "customization-options-for-my-store-design"
```

Comment:
Nit: can we use American English instead 😅 

```suggestion
        case customizationOptionForStoreDesign = "customization-options-for-my-store-design"
```

Reply:
😄  Done in https://github.com/woocommerce/woocommerce-ios/pull/10389/commits/f56ec682576ff0a4a4329f7b8011cfc2894cd1c3